src/Altinn.Apps/AppTemplates/AspNet/App/config/authorization/policy.xml

```diff
@@ -220,6 +220,55 @@
       </xacml:AnyOf>
     </xacml:Target>
   </xacml:Rule>
+  <xacml:Rule RuleId="urn:altinn:example:ruleid:6" Effect="Permit">
```

Comment:
don't we need a rule authorizing the app owner to read events as well?

Reply:
Yes, this rule does.

Reply:
Ah, didn't see the org tag there.. All good then!